### DISCUSSION TITLE
Touch Pin

### DISCUSSION THREAD

**Discussion by denizsahin7744** [HomeSpan/HomeSpan#849] [Q&A]
Hi i saw homespan support touch pins of esp32 on readme file but i couldn't find how to use it. Can someone help me ?

**Comment by SWills73** [comment]
I’ve just implemented that with a rope water leak sensor - use touchread(GPIOPin) to read the sensor state. One end of the sensor to GND, the other to the GPIO pin

**Reply by denizsahin7744** [reply]
i know its one of the solutions but i have copper plates for touch pins

**Reply by HomeSpan** [reply]
See the Tutorial 15 - RealPushButtons for info on using touch pins.
